analysis/geospatial_check.sql

```diff
@@ -0,0 +1,27 @@
+DROP IF EXISTS geospatial_check_table;
+CREATE TABLE IF NOT EXISTS geospatial_check_table(
+    v character varying,
+    result character varying
+);
+
+DELETE FROM geospatial_check_table
+WHERE v = :'ccp_v';
+
+INSERT INTO geospatial_check_table (
+select :'ccp_v' as v, 	 
+        jsonb_agg(t) as result
+from (
+    select jsonb_agg(json_build_object('projectid', tmp.projectid, 'magencyacr',tmp.magencyacr, 
+                              'typecatego', tmp.typecatego)) as values, 
+                              'projects_not_within_NYC' as field
+	from (SELECT a.projectid, a.magencyacr, a.typecatego
+          FROM cpdb_dcpattributes_poly a, 
+               (SELECT ST_Union(geom) geom
+               FROM borough_boundaries_wi) combined 
+          WHERE NOT ST_WITHIN(a.geom, combined.geom)) tmp
+    union 
+    select jsonb_agg(tmp.admin_boundary_type)as values,'unique_admin_boundary_type' as field 
```

Comment:
Hey @Oysters1874 let's find sometime to talk about the implementation for this check. When drafting the issue, I was thinking that this would be a check comparing previous version to the current version and therefore might be done by reading in the final tables directly from DO on the app side.